tests/zfs-tests/include/libtest.shlib

```diff
@@ -2894,7 +2894,7 @@ function labelvtoc
 
 #
 # check if the system was installed as zfsroot or not
-# return: 0 ture, otherwise false
+# return: 0 if true, otherwise false
```

Comment:
This still has a type mismatch (0 vs false). How about "return: 0 if true, non-zero if false" or "return 0 if zfsroot, non-zero if not".

Reply:
You guys tell me what you prefer.

Reply:
```suggestion
# return: 0 if zfsroot, non-zero if not
```

Reply:
I like the suggestion from @freqlabs

Reply:
> I like the suggestion from @freqlabs

Done!